packages/backend-defaults/src/entrypoints/auditor/Auditor.ts

```diff
@@ -0,0 +1,310 @@
+};
+
+/** @public */
+export type AuditorEventStatus<TError extends Error = Error> =
+  | { status: 'initiated' }
+  | { status: 'succeeded' }
+  | ({
+      status: 'failed';
+    } & ({ error: TError } | { errors: TError[] }));
+
+/**
+ * Options for creating an auditor event.
+ *
+ * @public
+ */
+export type AuditorEventOptions<TMeta extends JsonObject> = {
+  /**
+   * Use kebab-case to name audit events (e.g., "user-login", "file-download").
+   *
+   * The `pluginId` already provides plugin/module context, so avoid redundant prefixes in the `eventId`.
+   */
+  eventId: string;
+
+  severityLevel?: AuditorEventSeverityLevel;
+
+  /** (Optional) The associated HTTP request, if applicable. */
+  request?: Request<any, any, any, any, any>;
+
+  /** (Optional) Additional metadata relevant to the event, structured as a JSON object. */
+  meta?: TMeta;
+} & AuditorEventStatus;
+
+/**
+ * Common fields of an audit event.
+ *
+ * @public
+ */
+export type AuditorEvent = [
+  eventId: string,
+  meta: {
+    severityLevel: AuditorEventSeverityLevel;
+    actor: AuditorEventActorDetails;
+    meta?: JsonObject;
+    request?: AuditorEventRequest;
+  } & AuditorEventStatus,
+];
+
+/** @public */
+export const defaultProdFormat = winston.format.combine(
+  winston.format.timestamp({
+    format: 'YYYY-MM-DD HH:mm:ss',
+  }),
+  winston.format.errors({ stack: true }),
+  winston.format.splat(),
+  winston.format.json(),
+);
+
+/**
+ * Adds `isAuditorEvent` field
+ *
+ * @public
+ */
+export const auditorFieldFormat = winston.format(info => {
+  return { ...info, isAuditorEvent: true };
+})();
+
+/** @public */
+export interface RootAuditorOptions {
+  meta?: JsonObject;
+  format?: Format;
+  transports?: winston.transport[];
+}
+
+/**
+ * A {@link @backstage/backend-plugin-api#AuditorService} implementation based on winston.
+ *
+ * @public
+ */
+export class DefaultAuditorService implements AuditorService {
+  private readonly impl: DefaultRootAuditorService;
+  private readonly auth: AuthService;
+  private readonly httpAuth: HttpAuthService;
+  private readonly plugin: PluginMetadataService;
+
+  private constructor(
+    impl: DefaultRootAuditorService,
+    deps: {
+      auth: AuthService;
+      httpAuth: HttpAuthService;
+      plugin: PluginMetadataService;
+    },
+  ) {
+    this.impl = impl;
+    this.auth = deps.auth;
+    this.httpAuth = deps.httpAuth;
+    this.plugin = deps.plugin;
+  }
+
+  /**
+   * Creates a {@link DefaultAuditorService} instance.
+   */
+  static create(
+    impl: DefaultRootAuditorService,
+    deps: {
+      auth: AuthService;
+      httpAuth: HttpAuthService;
+      plugin: PluginMetadataService;
+    },
+  ): DefaultAuditorService {
+    return new DefaultAuditorService(impl, deps);
+  }
+
+  private async log<TMeta extends JsonObject>(
+    options: AuditorEventOptions<TMeta>,
+  ): Promise<void> {
+    const auditEvent = await this.reshapeAuditorEvent(options);
+    this.impl.log(auditEvent);
+  }
```

Comment:
Please make the default behavior that this logs directly to the `logger` or `rootLogger` service and nothing else. Let me know if there are any conflicts or problems with overlapping meta fields when doing that and why can try to sort those out. I don't want to full additional Winston setup for the audit service, especially not for local development.